happytransformer/adaptors/__init__.py

```diff
@@ -0,0 +1,16 @@
+from .adaptor import Adaptor
+from .albert import AlbertAdaptor
+from .berts import BertAdaptor,DistilBertAdaptor,RobertaAdaptor
+
+ADAPTORS = {
+    'BERT':BertAdaptor(),
+    'DISTILBERT':DistilBertAdaptor(),
+    'ROBERTA':RobertaAdaptor(),
+    'ALBERT':AlbertAdaptor()
+}
+
+def get_adaptor(model_type:str)->Adaptor:
```

Comment:
Are you sure this goes in __init__.py along with AdAPTORS? Or should __init__.py just reference it?